gammapy/maps/base.py

```diff
@@ -771,6 +771,10 @@ def interp_by_pix(self, pix, interp=None, fill_value=None):
         """
         pass
 
+    def fill_events(self, events):
```

Comment:
I'm still 48:52 to not add `.fill_events()` and just use `Map.fill_by_coord(events.map_coord)` otherwise one could argue why we don't implement `.fill_bkg()`, `fill_psf()`, etc. as well...

Reply:
We do have this for PSF:
https://github.com/gammapy/gammapy/blob/de8e1180184d3403484902cd612d655c0b593f5b/gammapy/maps/wcsnd.py#L615

For BKG - there's no simple object such that `map.fill_bkg(bkg)` would make sense.